proto/toit/api/device.proto

```diff
@@ -79,6 +80,27 @@ message DeviceEvent {
   Type type = 4;
   string msg = 5;
   bytes data = 6;
+  Initiater initiater = 7;
+}
+
+message Initiater {
+
+  message Device {
+    bytes device_id = 1;
+  }
+
+  message Console {}
+
+  message User {
+    bytes user_id = 1;
+  }
+
+  oneof initiater {
+    google.protobuf.Empty unknown = 1;
```

Comment:
remove this - if we have an unknown scenario we need to better at specifying it.